go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -181,6 +181,8 @@ func TestPlan(t *testing.T) {
 	testFile(t, "union_cases.txt", testOutputTempDir, vschemaWrapper)
 	testFile(t, "transaction_cases.txt", testOutputTempDir, vschemaWrapper)
 	testFile(t, "lock_cases.txt", testOutputTempDir, vschemaWrapper)
+	testFile(t, "ddl_cases_no_default_keyspace.txt", testOutputTempDir, vschemaWrapper)
+	testFile(t, "show_cases_no_default_keyspace.txt", testOutputTempDir, vschemaWrapper)
```

Comment:
Was there a need to create these two new files? The existing tests were to check even having default keyspace ddl and show queries are routes to desired keyspace.

Reply:
if the default keyspace is set, then only that is checked for tables without qualifiers but some of these tests relied on vtgate finding the table without adding the qualifier.